samples/GraphQL.DataLoader.Example/DealershipDbContext.cs

```diff
@@ -0,0 +1,72 @@
+﻿using DataLoaderGql.Types;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataLoaderGql;
+
+public class DealershipDbContext : DbContext
+{
+
+    public DbSet<Car> Cars => Set<Car>();
+    public DbSet<Salesperson> Salespeople => Set<Salesperson>();
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        optionsBuilder.UseSqlite(new SqliteConnectionStringBuilder()
```

Comment:
`.UseSqlite` should be on new line when chained with additional commands like `.UseSeeding`